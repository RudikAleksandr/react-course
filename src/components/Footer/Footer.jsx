import React from 'react';
import { Row, Nav, Col } from 'react-bootstrap';
import cn from './Footer.module.css';

const Footer = () => (
  <Col>
    <Row className={`justify-content-center align-items-center ${cn.footer}`}>
      <Nav.Link className={cn.netflixLink} href="/home">
        <b>netflix</b>
        roulette
      </Nav.Link>
    </Row>
  </Col>
);

export default Footer;
