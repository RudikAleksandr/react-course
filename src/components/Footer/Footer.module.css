.footer {
  background-color: #424242;
  height: 70px;
}

.netflixLink {
  color: #f65261;
  font-size: 1.5rem;
}

.netflixLink:hover {
  color: #f65261;
}
