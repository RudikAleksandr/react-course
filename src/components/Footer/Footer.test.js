
import { render } from '@testing-library/react';
import Footer from './Footer';

describe('Footer component', () => {
  test('render Footer snapshot', () => {
    const { asFragment } = render(<Footer />);

    expect(asFragment(<Footer />)).toMatchSnapshot();
  });
});