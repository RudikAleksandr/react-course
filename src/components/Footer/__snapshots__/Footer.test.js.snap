// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component render Footer snapshot 1`] = `
<DocumentFragment>
  <div
    class="col"
  >
    <div
      class="justify-content-center align-items-center footer row"
    >
      <a
        class="netflixLink nav-link"
        href="/home"
      >
        <b>
          netflix
        </b>
        roulette
      </a>
    </div>
  </div>
</DocumentFragment>
`;
